examples/master.js

```diff
@@ -346,7 +353,7 @@ function onPeerConnectionFailed() {
         if (!master.websocketOpened) {
             console.log('[MASTER] Websocket is closed. Reopening...');
             master.signalingClient.open();
-        } else if (getFormValues().ingestMedia) {
+        } else if (master.streamARN) {
```

Comment:
Shouldn't this be checking the form value of the checkbox for media server?

Reply:
No. We want to check if we are in the media service mode.

`formValues.ingestMedia` + `DescribeMediaStorageConfiguration=ENABLED` => `master.streamARN` (which represents that we are in media service mode)

Reply:
Looking at it, we already check for master.streamARN on L337, so this `else if` statement can be simplified to an `else`.

Reply:
We can do that later since this fix is urgent